docs/sla.md

```diff
@@ -0,0 +1,93 @@
+# Service Level Agreement(SLA)
+
+Service Level Agreement is available to Premium Customers only.
+
+
+## Definitions
+
+* "Self-service Console" provides access to a self-service console at https://console.cdnetworks.com. The console provides a way to configure the service through the Web UI or API. 
+* "Self-service Documentation" is a self-service documentation portal at https://docs.cdnetworks.com and can be accessed at any time.
+* "Email Support" is a way to obtain help through one of the dedicated email addresses that can be found at https://www.cdnetworks.com/contact
+* "Phone Support" is a way to obtain help through a dedicated phone line. Toll-free phone numbers can be found at https://www.cdnetworks.com/contact.
+* "Direct Tickets Support" is a way to obtain help through a dedicated ticket tracker, where all requests can be managed and handled easily.
+* "Business Hours" are working hours (9AM to 6PM, local time) in California, London, and Beijing.
+* "Business Days" are Monday through Friday at the customer location. 
+* "Urgent requests" are for issues that impact 50% or more of the customer’s users.
+* "High-priority requests" are for issues that impact less than 50% of the customer’s users.
+
+## Standard Support
+
+Trial and Paid customers can use an online self-service resource (console/documentation) provided by the Company and set up services by themselves. If any questions or issues arise, customers can contact Customer Support by email. A response will be available within one business day.
+
+## Premium Support
+
+Premium customers can use an online self-service resource (console/documentation) provided by the Company. They can set up services by themselves or use dedicated engineering resources allocated to them by the Company. If any questions or issues arise, premium customers can contact Customer Support by phone, email, and/or Ticket Tracker.
+
+The Company’s Support Engineers will adhere to the following rules:
+
+Urgent requests:
+1. Engineers will respond to customer within 15 minutes of receipt, start  troubleshooting, and update the status every 30 minutes;
+2. Engineers will work closely with customers 24x7  until the issue is resolved;
+3. Engineers will issue analysis reports within 24 hours after the incident is resolved.
+
+
+High-priority requests:
+1. Engineers will respond to tickets within 2 hours of receipt;
+2. Engineers will start troubleshooting and update the process every 4 hours;
```

Comment:
What is "the process"?   In #1, you refer to "tickets". It's better to stick to one phrase than switching to something new.